index.html

```diff
@@ -0,0 +1,107 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Technical Document</title>
+  <link rel="stylesheet" href="./css/style.css">
+</head>
+<body>
+  <div class="main-body">
+  <main>
+    <div id="content">
+    <h1 id = "section1">Introduction</h1>
+    <p>
```

Comment:
please do not add empty spaces while declaring ids and classes. It should be id="section1".

Reply:
@BenMukebo Hi Ben, I have a problem. In my dashboard, the status was "changes required" even though you approved this. When I submitted the link again for final project submission, it says review pending. Please assist.
ie- This project is for my "review" Review. not code review.